lms/djangoapps/instructor/views/api.py

```diff
@@ -221,6 +223,25 @@ def wrapped(*args, **kwargs):  # pylint: disable=missing-docstring
     return decorator
 
 
+def require_sales_admin(func):
```

Comment:
I wonder if it would be worth generalizing this even more?  I can imagine a decorator in `student.roles` that accepts a `CourseRole` subclass and a course ID.  You could then redefine this decorator in terms of the more general decorator.